src/Resources/config/shop_routing.yml

```diff
@@ -8,3 +8,9 @@ sylius_refund_plugin_shop_order_credit_memos_partial:
             repository:
                 method: findByOrderNumber
                 arguments: $orderNumber
+
+sylius_refund_shop_credit_memo_download:
+    path: /credit-memos/{id}/download
+    methods: [GET]
+    defaults:
+        _controller: Sylius\RefundPlugin\Action\DownloadCreditMemoAction
```

Comment:
I think this is a possible security issue - we're using the same action for Admin panel and Customer account, therefore it's possible for a Customer to access any credit memo in the system :( We should create a new action that would check the possibility to download credit memo... There is not so much logic to duplicate :)